HTML pages/about.html

```diff
@@ -0,0 +1,179 @@
+                <a href="home.html" class="link-portfolio"> Home </a>
+              </li>
+              <li class="menu-item">
+                <a href="about.html" class="link-about"> About </a>
+              </li>
+              <li class="menu-item">
+                <a href="/" class="link-contact"> Contact </a>
+              </li>
+            </ul>
+          </div>
+          <div class="reg-btn">
+            <button type="button">RG Gymshrk</button>
+          </div>
+        </nav>
+      </div>
+    </header>
+
+    </header>
+
+    <!-- First Section -->
+    <section class="about">
+       <div class="header-nav-mobile">
+        <button type="button" class="menu-bar-mobile" class="">
+          <img src="/assets/Icon - Menu.svg" alt="mobile menu bar icon" />
+        </button>
+
+        <!-- close button of mobile menu -->
+
+        <button type="button" class="menu-bar-close">
+          <img src="/assets/Icon-Cancel.svg"  alt="Close button">
+        </button>
+        
+        <!-- overlay background mobile menu function -->
+        <div class="overlay">
+
+        <nav class="navigation-mobile">
+          <div class="navigation">
+            <ul class="menu-list-mobile">
+              <li class="menu-item-mobile">
+                <a href="home.html" class="link-home"> Home </a>
+              </li>
+              <li class="menu-item-mobile">
+                <a href="about.html" class="link-about"> About </a>
+              </li>
+              <li class="menu-item-mobile">
+                <a href="/" class="link-contact">
+                  Contact
+                </a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+      </div>
+      </div>
+
+        <div class="flex-about">
+          <h3 class="about-first-title">BEST GYM CENTER</h3>
+          <h1 class="about-second-title">GROW YOUR STRENGTH WITH OUR TRAINERS</h1>
+        </div>
+        <div class="info-box-about">
+        <p class="box-brief">Contrary to popular belief, Lorem Ipsum is not simply random text. 
+           It has roots in a piece of classical Latin literature from 45 BC, making it over 
+           2000 years old. Richard McClintock,
+           a Latin professor at Hampden-Sydney College in Virginia</p>
+           <p class="help-text">Please contact us per Email for any further questions about GYMSHRK</p>
+           <a href="gmail.com" class="info-email">info@gymshrk.com</a>
+           </div>
+    </section>
+
+    <!-- Second section -->
+
+<section class="about-second">
+    <div class="about-second-container">
+        <h1 class="com-title">GYMSHRK Weight Lifting Competition Winner</h1>
+        <span class="com-title-dash"></span>
+        <p class="com-description">Contrary to popular belief, Lorem Ipsum is not simply random text. 
+           It has roots in a piece of classical Latin literature from 45 BC</p>
+        <div class="com-winner">
+            <img src="/assets/img/about/PNG.png" alt="">
```

Comment:
Weldone implementing some of the changes as suggested by the previous reviewer, please as he hiiglighted, you still have some excess white space issues that makes your project looks disorganized, it will really be helpful if you remove the excess white spaces and make your project, look as close as possible to the design given 😄 

Your design :

> ![image](https://github.com/Rifat8080/mahadi_capstone_project_1/assets/20537712/836f9102-5fc0-472e-8573-a43ee8d68a70)


> ![image](https://github.com/Rifat8080/mahadi_capstone_project_1/assets/20537712/c3f9c478-e75f-43a4-b54f-0fb4674832b1)`

Design given :
![image](https://github.com/Rifat8080/mahadi_capstone_project_1/assets/20537712/53fc23f2-3fb0-4106-bd66-b5dc4974b32d)